app/views/pages/main.html.haml

```diff
@@ -3,14 +3,14 @@
 
 %h2.ui.header transientBug
 %p
-  transientBug offers basic bookmark functionality, akin to pocket, pinboard, delicious and others, but with a few benifits:
+  transientBug offers basic bookmark functionality, akin to pocket, pinboard, delicious and others, but with a few benefits:
   %ul.list
     %li
       %b Cheap
-      If it works out well I might look into some low cost plans, but for now its free.
+      If it works out well I might look into some low cost plans, but for now its free. That being said, transientBug is currently in a closed, invite only registration right now, until it gets to a stable and complete feature-set that is ready for public use.
```

Comment:
This is a good thing to say, but you probably could just say:

For now it's free. transientBug is currently in a closed, invite only registration. When it gets to a stable and complete feature-set that is ready for public use, that all might change.